src/ir_Technibel.h

```diff
@@ -0,0 +1,151 @@
+// Copyright 2020 Quentin Briollant
+
+/// @file
+/// @brief Support for Technibel protocol.
+
+#ifndef IR_TECHNIBEL_H_
+#define IR_TECHNIBEL_H_
+
+#define __STDC_LIMIT_MACROS
+#include <stdint.h>
+#ifndef UNIT_TEST
+#include <Arduino.h>
+#endif
+#include "IRremoteESP8266.h"
+#include "IRsend.h"
+#ifdef UNIT_TEST
+#include "IRsend_test.h"
+#endif
+
+// Supports:
+//   Brand: TECHNIBEL,  Model: IRO PLUS
+
+// Ref:
+//
+
+// Kudos:
+//   : For the breakdown and mapping of the bit values.
```

Comment:
As you wrote it, and your names in the copyright, you can probably remove these comments. ;-)